components/forms/Button/Button.tsx

```diff
@@ -12,23 +11,14 @@ interface ButtonProps {
   destructive?: boolean;
   testid?: string;
   ariaProps?: React.AriaAttributes;
+  dataAttrName?: string;
```

Comment:
Given the `...props` we should be able to remove the `dataAttrName` and  `dataAttrValue`

Reply:
Removed!